### PR TITLE
https validation for testEndpoint

### PR DESCRIPTION
**Pull request by SagarikaKatikam**

### CODE REVIEW

File: packages/publisher-dapp/src/Pages/AiServiceCreation/Submit/SubmitForReview.js [hunk 1]
```diff
@@ -54,6 +55,14 @@ class SubmitForReview extends React.Component {
   validateDaemonEndpoint = async () => {
     const { serviceDetails } = this.props;
     const testEndPoint = serviceDetails.groups[0].testEndpoints;
+    if (isEmpty(testEndPoint)) {
+      return this.setState({
+        validateDaemonAlert: {
+          type: alertTypes.ERROR,
+          message: `The Ropsten end point can not be empty `,
```
Comment (Vivek205) [inline packages/publisher-dapp/src/Pages/AiServiceCreation/Submit/SubmitForReview.js:62]: remove the space between endpoint.
And message is a simple text. so replace \` with \"

File: packages/publisher-dapp/src/Pages/AiServiceCreation/Submit/SubmitForReview.js [hunk 2]
```diff
@@ -128,17 +137,28 @@ class SubmitForReview extends React.Component {
     }
   };
 
+  handleTEstEndpointValidation = value => {
```
Comment (Vivek205) [inline packages/publisher-dapp/src/Pages/AiServiceCreation/Submit/SubmitForReview.js:140]: use proper casing for the methods

File: packages/publisher-dapp/src/Pages/AiServiceCreation/Submit/SubmitForReview.js [hunk 3]
```diff
@@ -18,9 +18,10 @@ import { generateDetailedErrorMessageFromValidation } from "../../../Utils/valid
 import { ConfigurationServiceRequest } from "../../../Utils/Daemon/ConfigurationService";
 import ValidateConfig from "./ValidateConfig";
 import ValidationError from "shared/dist/utils/validationError";
+import isEmpty from "lodash/isEmpty";
 
 class SubmitForReview extends React.Component {
-  state = { daemonConfig: {}, alert: {}, validateDaemonAlert: {} };
+  state = { daemonConfig: {}, alert: {}, validateDaemonAlert: {}, testEndpointAlert: {} };
```
Comment (Vivek205) [inline packages/publisher-dapp/src/Pages/AiServiceCreation/Submit/SubmitForReview.js:24]: we don't want separate state for testEndpointAlert.

validateDaemonAlert is already being used for testEndpoint validation only, so we can use the same

File: packages/publisher-dapp/src/Pages/AiServiceCreation/Submit/validationConstraints.js [hunk 4]
```diff
@@ -4,6 +4,7 @@ export const submitServiceConstraints = {
   shortDescription: { presence: { allowEmpty: false } },
   longDescription: { presence: { allowEmpty: false } },
   contributors: { presence: { allowEmpty: false, message: "^Please provide the contributors details of the service" } },
+  testEndpoints: { validURL: { message: "Invalid Endpoint. Endpoint should start with Https" } },
```
Comment (Vivek205) [inline packages/publisher-dapp/src/Pages/AiServiceCreation/Submit/validationConstraints.js:7]: Make sure, adding this extra validation does not affect the continue and finish later

Comment (SagarikaKatikam) [reply packages/publisher-dapp/src/Pages/AiServiceCreation/Submit/validationConstraints.js:7]: We need it, as all the error boxes are getting  displayed when we have an error in one field.

**Pull request merged**